model/feature-flag/logs.yaml

```diff
@@ -1,11 +1,14 @@
 groups:
-  - id: log-feature_flag
```

Comment:
that's effectively the same change as made in the https://github.com/open-telemetry/semantic-conventions/pull/1440/files#diff-1db9b9bcfb009755e349125b55d3c945e2b5636723e91ee73ac4cd9184108036